packages/story-editor/src/components/canvas/rightClickMenu.js

```diff
@@ -89,19 +157,30 @@ const RightClickMenu = () => {
             isRTL={isRTL}
           >
             {rightClickMenuItems.map(
-              ({ label, shortcut, separator, ...buttonProps }) => (
+              ({
+                label,
+                shortcut,
+                separator,
+                subMenuItems,
+                ...buttonProps
+              }) => (
                 <Fragment key={label}>
                   {separator === 'top' && (
                     <ContextMenuComponents.MenuSeparator />
                   )}
-                  <ContextMenuComponents.MenuButton {...buttonProps}>
-                    {label}
-                    {shortcut && (
-                      <ContextMenuComponents.MenuShortcut>
-                        {shortcut.display}
-                      </ContextMenuComponents.MenuShortcut>
+                  <ButtonWrapper>
+                    {menuButtonRenderer({ label, shortcut, ...buttonProps })}
+                    {Boolean(subMenuItems?.length) && (
+                      <SubMenuWrapper
```

Comment:
💭 this needs an `aria-pressed` attribute. With using a nested context menu I believe this would still be true.

![image](https://user-images.githubusercontent.com/22185279/150138448-eda07f7c-e73b-423b-8ba7-df5799ee6bc7.png)

Reply:
The message says that `aria-pressed` is **not allowed though** not that it's needed. Will take a look.

Reply:
Hmm weird. Radix seems to match this implementation too:
![image](https://user-images.githubusercontent.com/22185279/150154703-5cdb93df-cfa4-4544-8594-9dd046afaeb5.png)

you're right, `aria-pressed` should not be needed